app/enterprise/1.3-x/kong-kubernetes/install.md

```diff
@@ -14,7 +14,7 @@ Before installing Kong for Kubernetes Enterprise, be sure you have the following
 - **Kubernetes cluster**: You can use [Minikube](https://kubernetes.io/docs/setup/minikube/) or a [GKE](https://cloud.google.com/kubernetes-engine/) cluster. Kong is compatible with all distributions of Kubernetes. 
 - **kubectl access**: You should have `kubectl` installed and configured to communicate to your Kubernetes cluster.
 - A valid Kong Enterprise License
-  * If you have a license, see [https://docs.konghq.com/enterprise/latest/deployment/licensing/](https://docs.konghq.com/enterprise/latest/deployment/licensing/)
+  * If you have a license, see "Step 1. Set Kong Enterprise License" below. 
```

Comment:
This will make it link to the step:

```suggestion
  * If you have a license, see ["Step 1. Set Kong Enterprise License"](#step-1-set-kong-enterprise-license) below. 
```